### PR TITLE
Update to okhttp 3.0.1 and fix docs

### PR DESCRIPTION
**Pull request by yairkukielka**
dependencies have been updated in the pom.xml file.
Also included the context object for the Picasso builder in the docs

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
   <groupId>com.jakewharton.picasso</groupId>
   <artifactId>picasso2-okhttp3-downloader</artifactId>
-  <version>1.0.2-SNAPSHOT</version>
+  <version>1.0.3-SNAPSHOT</version>
```
Comment (JakeWharton) [inline pom.xml]: This should not be changed.

**Comment by yairkukielka** [comment]
Ok, that change has been reverted. It's back to 1.0.2-SNAPSHOT

**Comment by JakeWharton** [comment]
Merged manually and released in 1.0.2.

**Comment by JakeWharton** [comment]
Thanks!

**Pull request closed without merging**